PWGEM/PhotonMeson/DataModel/gammaTables.h

```diff
@@ -411,6 +411,16 @@ using EMPrimaryElectronsFromDalitz = EMPrimaryElectronsFromDalitz_001;
 // iterators
 using EMPrimaryElectronFromDalitz = EMPrimaryElectronsFromDalitz::iterator;
 
+namespace v0photonsphiv
+{
+  DECLARE_SOA_INDEX_COLUMN(EMEvent, emevent); //!
+  DECLARE_SOA_COLUMN(PhiV, phiv, float);      //!
```

Comment:
The indentations here still need to go to pass the clang-format check.
We do not make indentations inside namespaces for some reason.